front/src/App.vue

```diff
@@ -1,19 +1,191 @@
 <template>
-  <div id="app">
+  <div id="app" class="font">
     <div>
       <router-link to="/">Home</router-link>
     </div>
     <router-view />
+    <div id="registration" class="blackout">
+      <div class="okno">
+        <div class="text">Создать аккаунт
+          <a href="#" class="close">
+            <el-button icon="el-icon-close" circle></el-button>
+          </a>
+        </div>
+        <div>
+          <el-form :model="ruleForm"
+                   :rules="rules"
+          >
+            <el-form-item prop="userName">
+              <el-input class="input" placeholder="Имя пользователя" v-model="ruleForm.userName">
+              </el-input>
+            </el-form-item>
+            <el-form-item prop="email">
+              <el-input class="input" placeholder="Email" v-model="ruleForm.email">
+              </el-input>
+            </el-form-item><el-form-item prop="pass">
+              <el-input class="input"
+                        type="password"
+                        placeholder="Пароль"
+                        v-model="ruleForm.password"
+              >
+              </el-input>
+            </el-form-item>
+          </el-form>
+        </div>
+        <div class="btn_padding">
+          <el-button type="primary" @click="Registration()">Зарегистрироваться</el-button>
+        </div>
+        <div class="entrance">
+          Уже есть аккаунт?
+          <a href="#">Вход</a>
+        </div>
+      </div>
+    </div>
+    <div id="created" class="blackout">
+      <div class="okno">
+        <div class="text">Создать аккаунт
+          <a href="#" class="close">
+            <el-button icon="el-icon-close" circle></el-button>
+          </a>
+        </div>
+        <div class="accountCreate font">
+          <el-button type="success" icon="el-icon-check" circle></el-button>
+          <p>Аккаунт создан!</p>
+          <p>Ты стал частью большого мира, где каждый может внести свою
+            лепту в то, чтобы процесс поиска фильма, сериала или мультфильма был проще и
+            увлекательней.
+          </p>
+          <p>Вперед за просмотрами, оценками и комментариями!</p>
+        </div>
+      </div>
+    </div>
+    <a href="#registration">
```

Comment:
why do we need this href?

Reply:
> why do we need this href?

link in the form of a button to go to registration